app/controllers/surveys_controller.rb

```diff
@@ -31,7 +31,13 @@ def create
 
     respond_to do |format|
       if @survey.save
-        format.html { redirect_to @survey, notice: 'Survey was successfully created.' }
+        format.html {
```

Comment:
Not the right approach here, how about a before action?

Reply:
We're also duplicating this code below so another sign that there is a better approach!

Reply:
Thanks @spencerldixon I will fix this.